stories/tree/tree.stories.js

```diff
@@ -1,897 +1,1019 @@
-                            <ul class="fd-menu__list fd-menu__list--no-shadow">
-                                <li class="fd-menu__item">
-                                    <a href="#" class="fd-menu__link">
-                                        <span class="fd-menu__title">Edit</span>
-                                    </a>
-                                </li>
-                                <li class="fd-menu__item">
-                                    <a href="#" class="fd-menu__link">
-                                        <span class="fd-menu__title">Lock</span>
-                                    </a>
-                                </li>
-                                <li class="fd-menu__item">
-                                    <a href="#" class="fd-menu__link">
-                                        <span class="fd-menu__title">Duplicate</span>
-                                    </a>
-                                </li>
-                                <li class="fd-menu__item">
-                                    <a href="#" class="fd-menu__link">
-                                        <span class="fd-menu__title">Delete</span>
-                                    </a>
-                                </li>
-                            </ul>
-                        </nav>
-                    </div>
-                </div>
+    <li role="treeitem" aria-level="1" class="fd-tree__item">
+        <div class="fd-tree__item-container">
+            <div class="fd-tree__content">
+                <span class="fd-tree__text">Level 1</span>
             </div>
         </div>
     </li>
 </ul>
 `;
 
-export const expandedTree = () => `
-<div class="fd-tree fd-tree--header">
-    <div class="fd-tree__row fd-tree__row--header">
-        <div class="fd-tree__col fd-tree__col--control">
-            <button class="fd-tree__control" aria-label="Expand all" aria-controls="tWsod582" aria-pressed="false"></button>
-            Column Header
-        </div>
-        <div class="fd-tree__col">
-            Column Header
-        </div>
-        <div class="fd-tree__col">
-            Column Header
-        </div>
-        <div class="fd-tree__col">
-            Status
-        </div>
-        <div class="fd-tree__col fd-tree__col--actions">
+treeWithExpandedLevel3AndHighlightIndicator.parameters = {
+    docs: {
+        iframeHeight: 500,
+        storyDescription: `
+Indentations are calculated with 0.5rem up to level 6.
+When Level 3 is expanded to show Level 4:
+
+- Level 2 indents by 0.5rem
+- Level 3 indents by 1rem
+- Level 4 indents by 1.5rem
+
+This indentation repeats when level 4 and 5 are triggered.
+
+- Level 5 indents by 2rem
+- Level 6 indents by 2.5rem
+
+Tree items can have highlight-indicators. To apply a highlight indicator use the <code class="docs-code">.has-highlight-indicator--*</code> modifier class on the <code class="docs-code">.fd-tree\\_\\_item-container</code> element. 
```

Comment:
I would just remove the "-" in highlight-indicators in the first sentence, so that the term is not differentiated from the second time its mentioned in the next sentence.
Both should be "highlight indicator" (no hyphen) 😄